helm/cas-provision/templates/job-runner/jobRunnerServiceAccount.yaml

```
@@ -0,0 +1,10 @@
+{{ if eq .Release.Namespace .Values.serviceAccounts.namespace }}
```

This service account should be in any `airflow` namespace. Something like this might work?
```suggestion
{{ if eq template "namespacePrefix" .  .Values.namespacePrefixes.airflow }}
```